BaristaSwift/Common/Common/Base/Store.swift

```diff
@@ -213,4 +204,10 @@ public class Store<objectType: StoreProtocol> {
         }
         return objectType.from(results)
     }
+    
+    enum StoreErrors : Error {
```

Comment:
I have errors in multiple classes, maybe I should centralize them?

Reply:
Sounds like a good idea.